modules/distr-tracing-rn-technology-preview.adoc

```diff
@@ -18,6 +18,25 @@ Technology Preview features are not supported with Red Hat production service le
 For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 ====
 
+== {DTProductName} 2.8.0 Technology Preview
+
+This release introduces support for {TempoName} as a Technology Preview feature for {DTProductName}.
+The feature uses version 0.1.0 of {TempoName} and version 2.0.1 of the upstream {TempoShortName} components.
+
+You can use {TempoName} to replace Jaeger so that you can use S3-compatible storage instead of ElasticSearch.
+Most users who use {TempoName} instead of Jaeger will not notice any difference in functionality because the Operator supports the same OLTP tracing protocol as Jaeger and uses the same user interface.
```

Comment:
> because the Operator supports the same OLTP tracing protocol

This is misleading:
* It's not OLTP but OTLP. 
* The operator does not support OTLP but the underlying product/instance/operator supports it - I would rewrite it to: `because the {TempoShortName}` supports the same ingestion and query protocols as Jaeger and uses the same user interface.

Reply:
@pavolloffay - Revised per your suggestion. PTAL. Thanks!